includes/Classifai/Admin/SavePostHandler.php

```diff
@@ -186,6 +187,26 @@ public function show_error_if() {
 		}
 	}
 
+	/**
+	 * Get the post statuses allowed to classify content
+	 *
+	 * @since 1.7.1 @TODO: Ensure this is updated to the version
+	 * @hook classifai_post_statuses
+	 *
+	 * @param string $post_type     The current post type.
+	 * @param int    $post_id       The current post ID.
+	 *
+	 * @return array $post_statuses The filtered array of post statuses.
+	 */
+	public function get_post_statuses( $post_type, $post_id ) {
+		// Defaults to published content only
+		$post_statuses = [ 'publish' ];
+
+		$post_statuses = apply_filters( 'classifai_post_statuses', $post_statuses, $post_type, $post_id );
```

Comment:
Should have a docblock on this filter. See other filters in this same file for examples

Reply:
@dkotter I have added docblocks to the filters (both of them).